ajax/libs/just-animate/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "just-animate",
-  "filename": "just-animate-animations.min.js",
+  "filename": "just-animate-all.min.js",
```

Comment:
For some reason I thought it would auto pull from NPM :)  I'll amend this with the 1.0.4 files